superset-frontend/packages/superset-ui-demo/storybook/stories/superset-ui-style/Theme.stories.tsx

```diff
@@ -25,23 +25,96 @@ export default {
 
 export const ThemeColors = () => {
   const { colors } = supersetTheme;
-  return Object.keys(colors).map(collection => (
+
```

Comment:
This change is improving the `Theme Colors` storybook as I kind of needed that to see throught his PR. Not directly related to the PR but helpful overall.